docs/harvesting.md

```diff
@@ -0,0 +1,222 @@
+# Harvesting
+
+Harvesting is the process of fetching of automatically remote metadata (ie. from other data portals or not)
+and store them into udata for being able to search them.
```

Comment:
`Harvesting is the process of automatically syncing remote metadata (i.e. from other data portals or not) and storing them into udata to be able to find remote data.`